views/style/style.css

```diff
@@ -0,0 +1,8981 @@
+/*!
```

Comment:
prefer libraries in separate files, or *better yet*, added as a dependency in package.json